### PR TITLE
Change tab to space in SSHFP requests

### PR DESCRIPTION
**Pull request by KalleDK**
Cloudflare uses space and not tab when you search for SSHFP records

##### SUMMARY

Fixes: #7652 

Cloudflare does not use tabs but space in SSHFP records when you search for them

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

cloudflare_dns

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by KalleDK** [comment]
> Thanks for your contribution! Can you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment)? Thanks.

I hope I've done it correct

File: changelogs/fragments/7653-fix-cloudflare-lookup.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - cloudflare_dns - fix cloudflare lookup of SHFP records in ``cloudflare_dns`` module (https://github.com/ansible-collections/community.general/issues/7652).
```
Comment (felixfontein) [inline changelogs/fragments/7653-fix-cloudflare-lookup.yml:2]: Just a small improvement - it's already clear from the `cloudflare_dns -` at the beginning that this is about the cloudflare_dns module, so no need to repeat it. Also Cloudflare itself is spelled with a capital C.
```suggestion
  - cloudflare_dns - fix Cloudflare lookup of SHFP records (https://github.com/ansible-collections/community.general/issues/7652).  
```

**Comment by felixfontein** [comment]
I'll merge this tomorrow if nobody objects.

**Comment by felixfontein** [comment]
@KalleDK thanks for your contribution!

**Pull request merged**